charms/istio-pilot/src/charm.py

```diff
@@ -124,7 +126,7 @@ def handle_default_gateway(self, event):
         )
         self._resource_handler.apply_manifest(manifest)
 
-        # Update the ingress resources as they rely on the default_gateway
+        # Update the ingress objects as they rely on the default_gateway
```

Comment:
We had an agreement to call them `resources` instead of `objects`. See [this](https://github.com/canonical/istio-operators/commit/312b6d027317f2dab2cf6947150ef628f2cac270) commit.